deploy/example_catalog/cr-app-tensorflow-notebook.json

```diff
@@ -0,0 +1,66 @@
+{
+    "apiVersion": "kubedirector.bluedata.io/v1alpha1",
+    "kind": "KubeDirectorApp",
+    "metadata": {
+        "name" : "jupyter-notebook"
+    },
+
+    "spec" : {
+        "systemdRequired": true,
+        "config": {
+            "roleServices": [
+                {
+                    "serviceIDs": [
+                        "ssh",
+                        "jupyter-nb"
+                    ],
+                    "roleID": "controller"
```

Comment:
As discussed, lets add worker role with no running service for people to be able to submit distributed TF jobs